plutus-core/plutus-core/src/PlutusCore/Pretty/PrettyConst.hs

```diff
@@ -97,11 +98,8 @@ instance Show a => DefaultPrettyBy ConstConfig (PrettyAny a) where
     defaultPrettyBy     _ = pretty . show @a   . coerce
     defaultPrettyListBy _ = pretty . show @[a] . coerce
 
-prettyConst :: PrettyConst a => a -> Doc ann
-prettyConst = prettyBy ConstConfig
-
-displayConst :: forall str a. (PrettyConst a, Render str) => a -> str
-displayConst = render . prettyConst
+prettyConst :: PrettyConst a => RenderContext -> a -> Doc ann
+prettyConst = prettyBy . ConstConfig
```

Comment:
Added `RenderContext`, because it does matter: sometimes we want parens around `list integer` for example (in the classic pretty-printer) and sometimes we don't (in the readable one occasionally or in error messages).